tcs-service/src/test/java/com/transformuk/hee/tis/tcs/service/service/impl/CurriculumMembershipServiceTest.java

```diff
@@ -107,4 +115,119 @@ void shouldThrowExceptionWhenPmNotFound() {
     assertThrows(NoSuchElementException.class,
         () -> cmService.save(dto));
   }
+
+  @Test
+  void shouldPatchCmDto() throws MethodArgumentNotValidException, NoSuchMethodException {
+    CurriculumMembershipDTO dto = createDto(CURRICULUM_ID, PM_UUID, START_DATE, END_DATE);
+    dto.setId(CURRICULUM_ID);
+
+    CurriculumMembership cm = new CurriculumMembership();
+    cm.setId(CURRICULUM_ID);
+    cm.setCurriculumStartDate(START_DATE);
+    cm.setCurriculumEndDate(END_DATE);
+    cm.setCurriculumId(CURRICULUM_ID);
+
+    ProgrammeMembership pm = new ProgrammeMembership();
+    pm.setUuid(PM_UUID);
+
+    CurriculumMembership returnedCm = new CurriculumMembership();
+    returnedCm.setId(CURRICULUM_ID);
```

Comment:
This line needs to be removed as you again set the id on line 138 (the 4th line below).